tests/unit/components/DocumentationTopic/TopicsLinkCardGrid.spec.js

```diff
@@ -56,11 +56,11 @@ describe('TopicsLinkCardGrid', () => {
       expect(pager.exists()).toBe(true);
       expect(pager.props('pages').length).toBe(2);
 
-      // 10 items => 3 pages at medium breakpoint (4 links per page)
+      // 10 items => 2 pages at large breakpoint (6 links per page)
```

Comment:
This comment should probably still say `medium`?

Reply:
Done! Good catch ;) https://github.com/apple/swift-docc-render/pull/787/commits/e841161bde0f5b83f612ccc0aa34e14b72354044